main.go

```diff
@@ -199,6 +198,7 @@ func (w *verboseGlogWriter) Write(p []byte) (n int, err error) {
 }
 
 func main() {
+	flag.Parse()
```

Comment:
This is to help with testing:
https://github.com/onsi/ginkgo/issues/602#issuecomment-557313846